hca/upload_to_cloud.py

```diff
@@ -89,6 +89,8 @@ def upload_to_cloud(files, staging_bucket, replica, from_cloud=False):
                     "hca-dss-content-type": _mime_type(fh.raw.name)
                 }
 
+                raw_fh.close()
```

Comment:
It's a list of binary files, but I would more accurately characterize it as a list of binary file handles.  Binary files could be read as either binary file names or binary file handles.  I had to trace the caller to see how it was managed.